### PR TITLE
Onboard `network-ranges list` and `network-ranges describe` commands

### PR DESCRIPTION
**Pull request by DiogoFerrao**

### CODE REVIEW

File: internal/cmd/beta/network-area/network-ranges/describe/describe.go [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+func NewCmd(p *print.Printer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "describe",
+		Short: "Shows details of a network range in a STACKIT Network Area (SNA)",
+		Long:  "Shows details of a network range in a STACKIT Network Area (SNA).",
+		Args:  args.SingleArg(networkRangeIdArg, utils.ValidateUUID),
+		Example: examples.Build(
+			examples.NewExample(
+				`Show details of a network range with id "xxx" in a STACKIT Network Area with ID "yyy" in organization with ID "zzz"`,
```
Comment (GokceGK) [inline internal/cmd/beta/network-area/network-ranges/describe/describe.go:44]:
```suggestion
				`Show details of a network range with ID "xxx" in a STACKIT Network Area with ID "yyy" in organization with ID "zzz"`,
```

File: internal/cmd/beta/network-area/network-ranges/describe/describe.go [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+	return &model, nil
+}
+
+func buildRequest(ctx context.Context, model *inputModel, apiClient *iaas.APIClient) iaas.ApiGetNetworkAreaRangeRequest {
+	req := apiClient.GetNetworkAreaRange(ctx, *model.OrganizationId, *model.NetworkAreaId, model.NetworkRangeId)
+	return req
+}
+
+func outputResult(p *print.Printer, outputFormat string, networkArea *iaas.NetworkRange) error {
```
Comment (GokceGK) [inline internal/cmd/beta/network-area/network-ranges/describe/describe.go:111]:
```suggestion
func outputResult(p *print.Printer, outputFormat string, networkRange *iaas.NetworkRange) error {
```

Comment (GokceGK) [reply internal/cmd/beta/network-area/network-ranges/describe/describe.go:111]: affects rest of the function

Comment (DiogoFerrao) [reply internal/cmd/beta/network-area/network-ranges/describe/describe.go:111]: Oops, missed that one

File: internal/cmd/beta/network-area/network-ranges/describe/describe.go [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+		if err != nil {
+			return fmt.Errorf("marshal network area: %w", err)
+		}
+		p.Outputln(string(details))
+
+		return nil
+	case print.YAMLOutputFormat:
+		details, err := yaml.MarshalWithOptions(networkRange, yaml.IndentSequence(true))
+		if err != nil {
+			return fmt.Errorf("marshal network area: %w", err)
```
Comment (GokceGK) [inline internal/cmd/beta/network-area/network-ranges/describe/describe.go:124]:
```suggestion
			return fmt.Errorf("marshal network range: %w", err)
```

**Review by GokceGK — APPROVED** [review]

**Pull request merged**